arelle/plugin/validate/EDINET/PluginValidationDataExtension.py

```diff
@@ -233,8 +235,14 @@ def getManifestInstance(self, modelXbrl: ModelXbrl) -> ManifestInstance | None:
         return controllerPluginData.matchManifestInstance(modelXbrl.ixdsDocUrls)
 
     def hasValidNonNilFact(self, modelXbrl: ModelXbrl, qname: QName) -> bool:
-        requiredFacts = modelXbrl.factsByQname.get(qname, set())
-        return any(fact.xValid >= VALID and not fact.isNil for fact in requiredFacts)
+        return any(fact is not None for fact in self.iterValidNonNilFacts(modelXbrl, qname))
```

Comment:
nit: checking if valid facts are not none is unnecessary. 
```python
return any(self.iterValidNonNilFacts(modelXbrl, qname))
```

Reply:
I prefer that as well but it does trigger this `FutureWarning`, which we might not care about but I decided to avoid it in this case. I just saw that it's possible to silence/filter these warnings, so I suppose that is an option as well.
```
FutureWarning: Truth-testing of elements was a source of confusion and will always return True in future versions. Use specific 'len(elem)' or 'elem is not None' test instead.
```

Reply:
I guess just `True for fact ...` would be even better here... I even saw that elsewhere recently but didn't think of it. I'll update on another PR.